project/FrontEnd/collaborative_science_platform/lib/main.dart

```diff
@@ -35,42 +36,16 @@ class MyApp extends StatelessWidget {
         ChangeNotifierProvider<NodeProvider>(create: (context) => NodeProvider()),
         ChangeNotifierProvider<UserProvider>(create: (context) => UserProvider()),
         ChangeNotifierProvider<WorkspaceProvider>(create: (context) => WorkspaceProvider()),
+        ChangeNotifierProvider<QuestionAnswerProvider>(
+            create: (context) => QuestionAnswerProvider()),
       ],
-      //    child: MaterialApp(
```

Comment:
I think its okey to remove these commented lines however It's better not changing anything out of scope.